### ISSUE TITLE
moveTo() isn't working.

### ISSUE THREAD

**Issue by twjolson** [asweigart/pyautogui#99]
I am unsure how to troubleshoot this.

The aim of this script is to manually run a browser (any browser, any version) to create internet artifacts for analysis.

So far, I just have the script launching the browser, moving the mouse to the address bar, and typing in the string.

It works fine when I do this once.  But, when I try to do this twice, with a 15 second sleep between them, I get this error : 
![image](https://cloud.githubusercontent.com/assets/21031913/17656845/14d5c612-6284-11e6-8b0d-7c2f5658537c.png)

The visible behavior is that it moves the mouse to the address bar, selects current text, but doesn't delete it.

Attached is the .py file.  
[bstest.txt](https://github.com/asweigart/pyautogui/files/417584/bstest.txt)
Any help would be appreciated.

**Comment by hugoesb** [comment]
I can see you're running the script on Windows. I've made the Pull Request "By pass PermissionError [WinError 5] #94" suggesting by pass a permission issue on Windows when we try use some functions like "_sendMouseEvent(MOUSEEVENTF_LEFTDOWN, x, y)".

Try this in your code:

```
try: 
    pyautogui.click()
except PermissionError:
    pass    
```

**Comment by asweigart** [comment]
This was fixed (or at least, worked around) in b53b659
